ansible/roles/test/files/tools/loganalyzer/loganalyzer.py

```diff
@@ -428,32 +427,32 @@ def analyze_file_list(self, log_file_list, match_messages_regex, ignore_messages
     #---------------------------------------------------------------------
 
 def usage():
-    print 'loganalyzer input parameters:'
-    print '--help                           Print usage'
-    print '--verbose                        Print verbose output during the run'
-    print '--action                         init|analyze - action to perform.'
-    print '                                 init - initialize analysis by placing start-marker'
-    print '                                 to all log files specified in --logs parameter.'
-    print '                                 analyze - perform log analysis of files specified in --logs parameter.'
-    print '                                 add_end_marker - add end marker to all log files specified in --logs parameter.'
-    print '--out_dir path                   Directory path where to place output files, '
-    print '                                 must be present when --action == analyze'
-    print '--logs path{,path}               List of full paths to log files to be analyzed.'
-    print '                                 Implicitly system log file will be also processed'
-    print '--run_id string                  String passed to loganalyzer, uniquely identifying '
-    print '                                 analysis session. Used to construct start/end markers. '
-    print '--match_files_in path{,path}     List of paths to files containing strings. A string from log file'
-    print '                                 By default syslog will be always analyzed and should be passed by match_files_in.'
-    print '                                 matching any string from match_files_in will be collected and '
-    print '                                 reported. Must be present when action == analyze'
-    print '--ignore_files_in path{,path}    List of paths to files containing string. '
-    print '                                 A string from log file matching any string from these'
-    print '                                 files will be ignored during analysis. Must be present'
-    print '                                 when action == analyze.'
-    print '--expect_files_in path{,path}    List of path to files containing string. '
-    print '                                 All the strings from these files will be expected to present'
-    print '                                 in one of specified log files during the analysis. Must be present'
-    print '                                 when action == analyze.'
+    print('loganalyzer input parameters:')
```

Comment:
I suggest adding the following import.
```
from __future__ import print_function
```
With this import, `print` is a built-in function like `Python3` instead of something like a macro.

Reply:
Done.